### ISSUE TITLE
Log in Azure Active Directory link leads to 404 Page not found error page

### ISSUE THREAD

**Issue by markusleino** [JetBrains/teamcity-azure-active-directory#45]
When pressing Log in Azure Active Directory link in TeamCity login screen you get error page "404 The page is not found". Reason seems to be wrong URL. It points to TeamCity server instead of Microsoft login page. If you replace TeamCity server address with Microsoft login page address and keep rest of the URL same it works fine. 

Plugin version is SNAPSHOT-20180516102618

**Comment by opernes** [comment]
Same issue here, trying to open a case at jetbrains and will follow up here

**Comment by laurencee** [comment]
Yep same issue just starting occuring for us as well a few days ago

**Comment by laurencee** [comment]
Ok I eventually figured out the actual cause of this for us, the problem was our IIS Reverse Proxy had "Reverse rewrite host in response headers" option enabled by someone in ARR. 

After disabling that the redirect worked as expected so I'm guessing something similar is going on here for others where response header host values are being incorrectly rewritten.
